src/lib/comms/sol-socket-dtls-impl-tinydtls.c

```diff
@@ -306,10 +285,11 @@ read_encrypted(void *data, struct sol_socket *wrapped)
     session_t session = { 0 };
     uint8_t buf[DTLS_MAX_BUF];
```

Comment:
It is going to be fun times when testing DTLS sockets on small devices WRT memory usage :)